libs/libpcf/src/base/pcf2place.cpp

```diff
@@ -102,6 +104,19 @@ int pcf2place(const PcfData& pcf_data,
       continue;
     }
 
+    size_t lsb = int_pin.get_lsb();
+    auto itr = net_map.find(lsb);
+    if (itr == net_map.end()) {
+      net_map.insert({lsb, net});
+    } else {
+      VTR_LOG_ERROR(
+        "Illegal pin constraint: FPGA IO pin is assigned to two design pins: "
```

Comment:
Need a clear message here. 
```
Illegal pin constraint: Two nets '%s' and '%s' are mapped to the I/O pins '%s' which belong to the same coordinate (x=%d, y=%d, z=%d) !
```

Reply:
OK. I will modify code according to your comment.Thanks!